src/natcap/invest/urban_nature_access.py

```diff
@@ -91,7 +93,10 @@
             'name': 'greenspace demand per capita',
             'units': u.m**2,  # defined as m² per capita
             'expression': "value > 0",
-            'about': "",  # TODO, will know more about this when I implement.
+            'about': (
+                "The amount of greenspace that each resident should have. "
```

Comment:
Should this be: "The amount of greenspace that each resident should have access to."?

Reply:
Oh that's way better.  Can't guarantee ownership of greenspace these days!  Patched in 2faa9eaf